partcad/src/partcad/assembly_factory_assy.py

```
@@ -176,7 +177,9 @@ async def handle_node(self, assembly, node):
 
         # Check if this node is for an assembly
         if "links" in node and not node["links"] is None:
-            item = Assembly({"name": name})  # TODO(clairbee): revisit why node["links"]) was used there
+            item = Assembly(
+                assembly.project_name, {"name": f"{self.name}:{name}", "child": True}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)